mysql-test/mroonga/storage/collation/utf8mb4_uca1400_as_ci/r/french.result

```diff
@@ -0,0 +1,16 @@
+DROP TABLE IF EXISTS diaries;
+SET NAMES utf8mb4;
+CREATE TABLE diaries (
+content varchar(256) COLLATE utf8mb4_uca1400_as_ci,
+FULLTEXT INDEX (content)
+) DEFAULT CHARSET=utf8mb4;
+INSERT INTO diaries VALUES ('Le fruit est mûr.');
+INSERT INTO diaries VALUES ('LE FRUIT EST MÛR.');
+INSERT INTO diaries VALUES ('Le mur est blanc.');
+INSERT INTO diaries VALUES ('LE MUR EST BLANC.');
+INSERT INTO diaries VALUES ('L''homme n''est qu''un roseau.');
```

Comment:
Can we remove this?
It seems that `mûr`/`MÛR` and `mur`/`MUR` are enough.

Reply:
fix: [1cfe543](https://github.com/mroonga/mroonga/pull/910/commits/1cfe543adb87350bb27b474587bef76016a0de5e) Sure.